src/metadata.js

```diff
@@ -152,6 +152,19 @@ export function parquetMetadata(arrayBuffer, { parsers } = {}) {
           repetition_level_histogram: column.field_3.field_16.field_2,
           definition_level_histogram: column.field_3.field_16.field_3,
         },
+        geospatial_statistics: column.field_3.field_17 && {
+          bbox: column.field_3.field_17.field_1 && {
+            xmin: column.field_3.field_17.field_1.field_1,
+            xmax: column.field_3.field_17.field_1.field_2,
+            ymin: column.field_3.field_17.field_1.field_3,
+            ymax: column.field_3.field_17.field_1.field_4,
+            zmin: column.field_3.field_17.field_1.field_5,
+            zmax: column.field_3.field_17.field_1.field_6,
+            mmin: column.field_3.field_17.field_1.field_7,
+            mmax: column.field_3.field_17.field_1.field_8,
+          },
+          geospatial_types: column.field_3.field_17.field_2,
```

Comment:
We might want to decode this to a list of literal strings, like you did for `algorithm` in https://github.com/hyparam/hyparquet/commit/cc6cc86ba41816a3a9e6d72723a84d1ad19c5f8c

The list of values is here: https://github.com/apache/parquet-format/blob/master/Geospatial.md#geospatial-types

Reply:
not clear to me what the strings should be in that case? How do we distinguish 2D from 3D points? Also that list is not exhaustive, so I am inclined to leave these as wkb integer codes.